src/core/client/admin/routes/Moderate/ModerateCard/ToxicityLabel.tsx

```diff
@@ -0,0 +1,45 @@
+import { Localized } from "fluent-react/compat";
+import React, { FunctionComponent } from "react";
+
+import { Counter, Typography } from "coral-ui/components";
+
+import styles from "./ToxicityLabel.css";
+
+const ToxicityCounter: FunctionComponent<{ score: number }> = ({ score }) => (
+  <Counter>{Math.round(score * 100)}%</Counter>
+);
+
+const ToxicityLabel: FunctionComponent<{
+  score: number;
+  threshold: number;
+}> = ({ score, threshold }) => {
+  const counter = <ToxicityCounter score={score} />;
+
+  if (score > threshold) {
+    return (
+      <Localized id="moderate-toxicityLabel-likely" score={counter}>
+        <Typography
+          className={styles.root}
+          variant="bodyCopy"
+          color="errorDark"
+        >
+          Likely
+        </Typography>
+      </Localized>
+    );
+  } else if (score <= 0.5) {
```

Comment:
What if they set their toxicity score to 50? This 0.5 is hidden to the user, should it maybe be a ratio of what 0.5 is to the default 80 instead of a hard coded value?

i.e if user sets toxicity to 70 instead of 80.

50 is maybe when using default of 80.
Compute x for 80, which is 43.75.

What do you think of this?

Reply:
This was going off of the design. I'll follow up to verify this.

Reply:
Design has confirmed that this is working as intended.